### PR TITLE
Add via keymap for pancake

### PR DESCRIPTION
**Pull request by Spaceman**
## Description

- Adds via keymap for pancake keyboard
- Sets the vendor and product ID to more appropriate values

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/spaceman/pancake/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/spaceman/pancake/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/spaceman/pancake/config.h [hunk 2]
```diff
@@ -17,8 +17,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0xB195
+#define VENDOR_ID       0x5342 // SB 
+#define PRODUCT_ID      0x504b // PK
```
Comment (fauxpark) [inline keyboards/spaceman/pancake/config.h:21]:
```suggestion
#define PRODUCT_ID      0x504B // PK
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by Spaceman** [comment]
Some keymaps are incompatible with pancake as so they will not build hence why Travis CI build failed, it should be fine otherwise

**Review by ridingqwerty — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**